File: Examples/CSharp/Articles/PivotTablesAndPivotCharts/SettingPivotTableOption.cs
using System;
using System.IO;

using Aspose.Cells;
using Aspose.Cells.Pivot;

namespace Aspose.Cells.Examples.CSharp.Articles.PivotTablesAndPivotCharts
{
    public class SettingPivotTableOption
    {
        public static void Run()
        {
            //Source directory
            string sourceDir = RunExamples.Get_SourceDirectory();

            //Output directory
            string outputDir = RunExamples.Get_OutputDirectory();

            Workbook wb = new Workbook(sourceDir + "sampleSettingPivotTableOption.xlsx");

            PivotTable pt = wb.Worksheets[0].PivotTables[0];

            // Indicating if or not display the empty cell value
            pt.DisplayNullString = true;

            // Indicating the null string
            pt.NullString = "null";
            pt.CalculateData();

            pt.RefreshDataOnOpeningFile = false;

            wb.Save(outputDir + "outputSettingPivotTableOption.xlsx");

            Console.WriteLine("SettingPivotTableOption executed successfully.");
        }
    }
}

File: Examples/VisualBasic/Articles/PivotTablesAndPivotCharts/SettingPivotTableOption.vb
Imports Microsoft.VisualBasic
Imports System.IO
Imports Aspose.Cells
Imports System
Imports Aspose.Cells.Pivot

Namespace Articles.PivotTablesAndPivotCharts
    Public Class SettingPivotTableOption
        Public Shared Sub Run()
            ' ExStart:SettingPivotTableOption
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType)

            Dim wb As New Workbook(dataDir & Convert.ToString("input.xlsx"))

            Dim pt As PivotTable = wb.Worksheets(0).PivotTables(0)

            ' Indicating if or not display the empty cell value
            pt.DisplayNullString = True

            ' Indicating the null string
            pt.NullString = "null"
            pt.CalculateData()

            pt.RefreshDataOnOpeningFile = False

            wb.Save(dataDir & Convert.ToString("output_out.xlsx"))
            ' ExEnd:SettingPivotTableOption
        End Sub
    End Class
End Namespace